### DISCUSSION TITLE
Jerarquía semántica en documentación y componente Text

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#913] [Proposals]
Creéis que deberíamos añadir a la documentación de Brand Factory el componente <Text>? O mencionarlo en Typography?

Por otro lado, deberíamos añadir información sobre jerarquía semántica (h1-h6) en la documentación de diseño?

**Comment by aweell** [comment]
> Creéis que deberíamos añadir a la documentación de Brand Factory el componente ? O mencionarlo en Typography?

Quizás sería suficiente con poner como link text-presets en este callout?

![Screenshot 2022-11-18 at 11 05 08](https://user-images.githubusercontent.com/44420072/202676104-cba9a344-93f1-4b94-a55a-f96ea5d2f5e2.png)

> Por otro lado, deberíamos añadir información sobre jerarquía semántica (h1-h6) en la documentación de diseño?

Mientras exista una prop en el title que te deje indicar el nivel creo que es interesante añadir esa info en brand factory para que se entienda como usarla correctamente.

**Comment by AnaMontes11** [comment]
Lo primero, lo mencionaría en Typography. Con el callout que comenta Álex me parece que es suficiente.

En cuanto a la segunda, podríamos añadir en la página de BF del componente Title qué es la prop "Level" (h1-h6) y cómo usarlo. Lo veo necesario.
